docs/cluster-setup-guide/deploy-cluster/sysadmin-deploy-on-slurm/slurm-known-issues.rst

```diff
@@ -17,6 +17,14 @@ ignored.
 -  ``resources.priority``
 -  ``resources.weight``
 
+******************************************
+ Slurm and PBS Job Exit Code Report Issue
```

Comment:
IMO would be better to add to the existing "Slurm Known Issues" section, and duplicate in "PBS Known Issues"

Generally, we don't have per-issue headings.

Reply:
There is a section  "Slurm Known Issues" and there is a section  "Additional Known issues". What about adding it to the "Additional Known issues" without creating a new section "PBS Known issues"?

Reply:
That works for me also.

Reply:
Had trouble merging up to the master so created a new branch and new PR https://github.com/determined-ai/determined/pull/6233